hasher-matcher-actioner/webapp/src/components/ThreatExchangePrivacyGroupCard.jsx

```diff
@@ -40,12 +40,22 @@ export default function ThreatExchangePrivacyGroupCard({
           </Card.Subtitle>
           <Card.Body className="text-left">
             <Form>
+              {privacyGroupId === 258601789084078 ||
+              privacyGroupId === 303636684709969 ? (
+                <a
+                  href="https://lear.inrialpes.fr/~jegou/data.php#copydays"
+                  target="_blank"
+                  rel="noopener noreferrer">
+                  Copydays
```

Comment:
Can we make this say something like "This is a test collaboration. Download the test images here"